_includes/specials/taproot/en/06-multisignatures.md

```diff
@@ -0,0 +1,147 @@
+{% comment %}<!--
+  Using dump-multisigs from img/posts/2021-07-multisig-fungibility.gnuplot
+
+    for i in `seq 691039 692039` ; do dump-multisigs $i ; done > RESULTS
+    echo $( grep of RESULTS | wc -l ) / $( cat RESULTS | wc -l ) | bc -l
+-->{% endcomment %}
+In the 1,000 blocks received prior to this writing, 11% of all
+transaction inputs contained a multisig opcode.  Two of the largest and
+most immediate benefits of taproot will manifest if many of the users
+and services creating those transactions switch from multisig opcodes to
+scriptless [multisignatures][topic multisignature].
+
+The first major benefit will be a reduction in transaction size.
+Script-based multisigs increase in size as more keys and signatures are
+required, but multisignatures are a constant small size.  The smallest
+effective multisig policy (1-of-2) requires more space than a
+multisignature policy that can involve thousands of signers.  The
+decrease in size leads to direct reduction in fees for the
+multisignature users and an indirect reduction in fees for all users as
+the same amount of demand for confirmed transactions can be fulfilled
+using a smaller amount of block space.
+
+{:.center}
+![Plot showing the savings for multisignatures compared to multisig](/img/posts/2021-07-multisignature-savings.png)
+
+The second major benefit is improved privacy.  Each use of multisigs is
+distinctively recorded to the block chain where surveillance companies
+can use them to make informed guesses about the wallet history and
+current balance of individual users.  For example, looking at block
+692,039, we can distinguish not just the multisig spends from the
+single-sig spends but also distinguish between different set sizes and
+thresholds for the multisigs.
+
+{:.center}
+![Illustration of the lack of witness fungibility in current blocks](/img/posts/2021-07-multisig-unfungible.png)
+
+By comparison, a third party looking only at block chain data can't tell
+that a spender used a multisignature.  When a multisignature is used for
+a keypath spend, it is indistinguishable from single-sig spends.  If all
+single-sig and multisigs in the block above were switched to P2TR
+keypath spends, only a few exotic spends would be distinguishable by
+their scripts (and even some of those might be able to sometimes use
+keypath spends).
```

Comment:
The 'taproot assumption' is that _all_ outputs can be spent using a multisignature keypath spend:

> there exists some monotone function on plain public keys and nothing else that is sufficient to authorize a transaction

(https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-February/015700.html)

Can we replace "even some of those might be able to use keypath spends" to "and even those could use keypath spends in the best case" or similar.